components/analytics-core/org.wso2.carbon.analytics.datasource.core.test/src/main/java/org/wso2/carbon/analytics/datasource/core/AnalyticsDataServiceTest.java

```diff
@@ -1083,14 +1116,362 @@ public void testAnalyticsClusterManager() throws AnalyticsClusterException {
     }
 
-    @Test (enabled = false)
+    private boolean createNewTable(int tenantId, String tableName) throws AnalyticsException, InterruptedException {
+
+        this.cleanupTable(tenantId, tableName);
+        List<ColumnDefinition> columns = new ArrayList<>();
+        columns.add(new ColumnDefinition("Title", ColumnType.STRING, true, false));
+        columns.add(new ColumnDefinition("Author", ColumnType.FACET, true, false));
+        columns.add(new ColumnDefinitionExt("Published_Date", ColumnType.FACET, true, false));
+        columns.add(new ColumnDefinition("Count", ColumnType.INTEGER, true, true));
+        columns.add(new ColumnDefinition("Price", ColumnType.FLOAT, true, true));
+        columns.add(new ColumnDefinition("NoOfCopies", ColumnType.INTEGER, true, true));
+        columns.add(new ColumnDefinition("Rating", ColumnType.DOUBLE, true, true));
+
+        this.service.createTable(tenantId, tableName);
+        this.service.setTableSchema(tenantId, tableName, new AnalyticsSchema(columns, null));
+        List<Record> records = new ArrayList<>();
+        Map<String, Object> values = new HashMap<>();
+        values.put("Title", "The Sun Also Rise");
+        values.put("Author", "E.Hemingway");
+        values.put("Published_Date", "1926,08,09");
+        values.put("Count", "42");
+        values.put("Price", "45");
+        values.put("NoOfCopies", "5090");
+        values.put("Rating", "3.8");
+        Record record1 = new Record(tenantId, tableName, values);
+
+        values = new HashMap<>();
+        values.put("Title", "Divergent");
+        values.put("Author", "Veronica Roth");
+        values.put("Published_Date", "2011,04,25");
+        values.put("Count", "485");
+        values.put("Price", "28");
+        values.put("NoOfCopies", "6700000");
+        values.put("Rating", "4.2");
+        Record record2 = new Record(tenantId, tableName, values);
+
+        values = new HashMap<>();
+        values.put("Title", "Insurgent");
+        values.put("Author", "Veronica Roth");
+        values.put("Published_Date", "2011,05,01");
+        values.put("Count", "525");
+        values.put("Price", "35");
+        values.put("NoOfCopies", "11600000");
+        values.put("Rating", "4.8");
+        Record record3 = new Record(tenantId, tableName, values);
+
+        values = new HashMap<>();
+        values.put("Title", "Allegiant");
+        values.put("Author", "Veronica Roth");
+        values.put("Published_Date", "2016,02,16");
+        values.put("Count", "526");
+        values.put("Price", "30");
+        values.put("NoOfCopies", "11000000");
+        values.put("Rating", "4.4");
+        Record record4 = new Record(tenantId, tableName, values);
+
+        records.add(record1);
+        records.add(record2);
+        records.add(record3);
+        records.add(record4);
+        this.service.put(records);
+        this.service.waitForIndexing(DEFAULT_WAIT_TIME);
+
+
+        return true;
```

Comment:
Why return `true`? seems like this method always returns `true`. I don't see a necessity for that.